bin/fai-cd

```diff
@@ -188,10 +188,15 @@ create_squashfs_image() {
 # - - - - - - - - - - - - - - - - - - - - - - - - - -
 create_grub2_image() {
 
-    mkdir -p $tmp/boot
+	mkdir -p $tmp/boot
+	# Needs 'grub2-common' and 'fonts-dejavu-core' packages.
+	grub-mkfont -s 16 -o "$tmp/DejaVuSansMono.pf2" /usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf
 
     if [ -d $NFSROOT/usr/lib/grub/x86_64-efi ]; then
 	cp $grub_config $NFSROOT/tmp/grub.cfg
+	# Needs 'desktop-base' package.
+	cp /usr/share/images/desktop-base/desktop-grub.png $NFSROOT/tmp/background.png
```

Comment:
Maybe have the dependency on desktop-base more optional? Check if the file exists and only then copy it?